kitsune/sumo/static/sumo/js/upload.js

```diff
@@ -1,3 +1,15 @@
+function dialogSet(inner, title) {
+  var kbox = new KBox($('<p/>').text(inner), {
```

Comment:
why did you move this out?

Reply:
It needs to be outside the closure so I can expose it globally in webpack - there's a couple of other files which call it